apis/pkg/v1/package_types.go

```diff
@@ -89,4 +89,37 @@ type PackageStatus struct {
 	// will cause the package manager to check that the current revision is
 	// correct for the given package source.
 	CurrentIdentifier string `json:"currentIdentifier,omitempty"`
+
+	// AppliedImageConfigRefs records any image configs that were applied in
+	// reconciling this package, and what they were used for.
+	AppliedImageConfigRefs []ImageConfigRef `json:"appliedImageConfigRefs,omitempty"`
+
+	// Package is the name of the package that was used for version
+	// resolution. It may be different from spec.package if the package path was
+	// rewritten using an image config.
+	Package string `json:"package,omitempty"`
 }
+
+// ImageConfigRef is a reference to an image config that indicates how the
+// referenced image config was used by the package manager.
+type ImageConfigRef struct {
+	// Name is the name of the image config.
+	Name string `json:"name"`
+	// Reason indicates what the image config was used for.
+	Reason ImageConfigRefReason `json:"reason"`
+}
+
+// ImageConfigRefReason is a reason an image config was used.
+type ImageConfigRefReason string
+
+const (
+	// ImageConfigReasonPullSecretReference indicates an image config was used
+	// to configure the pull secret.
+	ImageConfigReasonPullSecretReference ImageConfigRefReason = "PullSecretReference"
+	// ImageConfigReasonVerifyImage inidcates an image config was used to
```

Comment:
nit: 

```suggestion
	// ImageConfigReasonVerifyImage indicates an image config was used to
```

same below.

Reply:
Good catch, fixed.